ios/Podfile

```diff
@@ -24,9 +24,11 @@ require_relative "../node_modules/react-native/scripts/react_native_pods"
 require_relative "../node_modules/@react-native-community/cli-platform-ios/native_modules"
 
 platform :ios, '12.4'
-# install! 'cocoapods', :deterministic_uuids => false
+install! 'cocoapods', :deterministic_uuids => false
 use_frameworks!
 
+production = ENV["PRODUCTION"] == "1"
```

Comment:
I wonder what this does.
Do you think we need to set this for release builds?

Reply:
i'll double check. i saw it being used in the use_react_native to trigger the use_flipper function...

Reply:
the only place this is used if to enable flipper / the react dev pods. in the use_react_native script
```
if flipper_configuration.flipper_enabled && !production
    pod 'React-Core/DevSupport', :path => "#{prefix}/"
    use_flipper!(flipper_configuration.versions, :configurations => flipper_configuration.configurations)
  end
```

i don't really know where ENV is defined though, i assume that we don't need to do anything extra if the CI will run?

Reply:
yes it should be good. Thanks.